test/unit/channel.js

```diff
@@ -569,6 +576,30 @@ describe('Channel _handleChannelEvent', function () {
 	});
 });
 
+describe('Uninitialized Channel', () => {
+	const user = { id: 'user' };
+	let client;
+	let channel;
+
+	beforeEach(() => {
+		client = new StreamChat('apiKey');
+		client.user = user;
+		client.userID = user.id;
+		client.userMuteStatus = (targetId) => targetId.startsWith('mute');
+		channel = client.channel('messaging', 'id');
+		channel.initialized = false;
+		channel.offlineMode = false;
+	});
+
+	it('returns 0 mentions in unread messages', () => {
+		expect(channel.countUnreadMentions()).to.eq(0);
```

Comment:
out of curiosity, isn't it better to return `-1` when the channel is not yet initialized? 

`0` can be easily confused with:
- I have no unread mentions
- My channel isn't initialized

Reply:
Yes, it gives more granularity to decision making. Also, it should not break any UI as until now, in this case the error would be thrown.